features/configuration/fail_if_no_examples.feature

```diff
@@ -0,0 +1,46 @@
+Feature: fail if no examples
+
+  Use the `fail_if_no_examples` option to make RSpec exit with exit status 1
```

Comment:
Technically RSpec will exit with the configured error failure status, so prehaps this should read:
```
to make RSpec exit with a failure status (by default 1)
```
which also avoids the double exit :)